ui/app/models/sync/destinations/gh.js

```diff
@@ -6,10 +6,25 @@
 import SyncDestinationModel from '../destination';
 import { attr } from '@ember-data/model';
 import { withFormFields } from 'vault/decorators/model-form-fields';
+import { withModelValidations } from 'vault/decorators/model-validations';
 
-@withFormFields()
+const validations = {
+  name: [{ type: 'presence', message: 'Name is required.' }],
+  accessToken: [{ type: 'presence', message: 'Access token is required.' }],
+  repositoryOwner: [{ type: 'presence', message: 'Repository owner is required.' }],
+  repositoryName: [{ type: 'presence', message: 'Repository name is required.' }],
+};
+const fields = ['name', 'accessToken', 'repositoryOwner', 'repositoryName'];
+
+@withModelValidations(validations)
+@withFormFields(fields)
 export default class SyncDestinationsGithubModel extends SyncDestinationModel {
-  @attr('string') accessToken;
-  @attr('string') repositoryOwner;
-  @attr('string') repositoryName;
+  @attr('string')
+  accessToken;
```

Comment:
It looks like the description is missing for `accessToken`
```suggestion
  @attr('string', { subText: 'The personal access token to authenticate to GitHub repository.')
  accessToken;
```

Reply:
good catch! thank you! 🕵️